FrontEnd/src/components/ProfileDetail/DetailedInfo/ReadMore.jsx

```diff
@@ -1,28 +1,42 @@
 import { useState } from 'react';
+import { Typography } from 'antd';
 import { PropTypes } from 'prop-types';
 import classes from './ReadMore.module.css';
 
+const { Paragraph } = Typography;
+
 const ReadMore = ({ children }) => {
   const text = children;
-  const [readMore, setReadMore] = useState(false);
+  const [ellipsis, setEllipsis] = useState(true);
 
   const toggleReadMore = () => {
-    setReadMore(!readMore);
+    setEllipsis(!ellipsis);
   };
-  const maxTextLength = 150;
 
-  const displayText = text && (readMore || text.length <= maxTextLength ? text : `${text.slice(0, maxTextLength)}...`);
+  const ellipsisSymbol = ellipsis ? (
+    <span className={classes['read-more-symbol']} onClick={toggleReadMore}>
+    читати далі
+    </span>
+  ) : null;
 
   return (
-    text ? (
-    <p className={classes['read-more']}>
-        {displayText}
-        {text.length > maxTextLength && (
-          <span onClick={toggleReadMore} className={classes['read-or-hide']}>
-            {!readMore ? 'читати далі' : 'приховати'}
-          </span>
-      )}
-    </p>) : null
+    <>
+      <Paragraph
+        className={classes['read-more']}
+        onClick={toggleReadMore}
+        ellipsis={
+          ellipsis
+            ? {
+                rows: 6,
+                expandable: true,
+                symbol: ellipsisSymbol,
+              }
```

Comment:
Could we move it to constant with a meaningful name?

Reply:
Should we do it if it is Ant Design component and its structure?

Reply:
Done